backend/interaction_service/models/message.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+
+/**
+ * Schema of a Like object in the database, like documents resides
+ * in 'likes' collection.
+ */
+module.exports.Message = mongoose.model(
```

Comment:
Nice to have in future: I didn't see that in our requirements but the customer might ask us to keep the messages encrypted in our database.